griptape/drivers/prompt_model/bedrock_claude_prompt_model_driver.py

```diff
@@ -14,7 +14,7 @@ class BedrockClaudePromptModelDriver(BasePromptModelDriver):
     top_k: int = field(default=250, kw_only=True, metadata={"serializable": True})
     _tokenizer: BedrockClaudeTokenizer = field(default=None, kw_only=True)
     prompt_driver: Optional[AmazonBedrockPromptDriver] = field(default=None, kw_only=True)
-    anthropic_version: str = field(default="bedrock-2023-05-31", kw_only=True, metadata={"serializable": True})
+    anthropic_version: str = "bedrock-2023-05-31"  # static string for AWS: https://docs.aws.amazon.com/bedrock/latest/userguide/model-parameters-anthropic-claude-messages.html#api-inference-examples-claude-multimodal-code-example
```

Comment:
If we're treating this as a class field, it should be `ANTHROPIC_VERSION = "bedrock-2023-05-31"`. But I think we should keep it as an attrs instance field for consistency with the other implementation.

Reply:
Which other implementation? I thought we discussed making this a static class field?

Reply:
Oh shoot, I didn't realize the file I was looking at. Ok then yeah let's just change it to `ANTHROPIC_VERSION = "bedrock-2023-05-31"` (no type) before the rest of the fields.